### ISSUE TITLE
usb 1-2 device descriptor read error on CML_HEL_RT5682.

### ISSUE THREAD

**Issue by aiChaoSONG** [thesofproject/sof-test#567]
On helios, some USB error messages caused false failure in kmod test.
```
[ 9210.249604] kernel: usb 1-2.4: device descriptor read/64, error -32
[ 9210.447526] kernel: usb 1-2.4: device descriptor read/64, error -32
[ 9210.730531] kernel: usb 1-2.4: device descriptor read/64, error -32
[ 9210.927570] kernel: usb 1-2.4: device descriptor read/64, error -32
[ 9211.340146] kernel: usb 1-2.4: device descriptor read/8, error -71
[ 9211.459192] kernel: usb 1-2.4: device descriptor read/8, error -32
[ 9211.667943] kernel: usb 1-2.4: device descriptor read/8, error -32
[ 9211.788330] kernel: usb 1-2.4: device descriptor read/8, error -32
[ 9212.099550] kernel: usb 1-2.4: device descriptor read/64, error -32
[ 9212.298523] kernel: usb 1-2.4: device descriptor read/64, error -32
[ 9212.586521] kernel: usb 1-2.4: device descriptor read/64, error -32
[ 9212.785426] kernel: usb 1-2.4: device descriptor read/64, error -32
```
```
[ 1903.733329] kernel: usb 1-2.4: device descriptor read/64, error -32
[ 1903.931479] kernel: usb 1-2.4: device descriptor read/64, error -32
[ 1904.211456] kernel: usb 1-2.4: device descriptor read/64, error -32
[ 1904.405589] kernel: usb 1-2.4: device descriptor read/64, error -32
[ 1905.017875] kernel: usb 1-2.4: device not accepting address 4, error -71
[ 1905.522086] kernel: usb 1-2.4: device not accepting address 4, error -71
[ 1905.743324] kernel: usb 1-2.4: device descriptor read/64, error -32
[ 1905.944366] kernel: usb 1-2.4: device descriptor read/64, error -71
[ 1906.228525] kernel: usb 1-2.4: device descriptor read/64, error -71
[ 1906.419646] kernel: usb 1-2.4: device descriptor read/64, error -71
```

**Comment by keqiaozhang** [comment]
Now CI detected some different errors of "usb-1.2.*" on different DUTs. Like :
```
[ 8956.787401] kernel: usb 1-2.3: can't restore configuration #1 (error=-71)
[ 8957.322112] kernel: usb 1-2.3: unable to read config index 0 descriptor/all
[ 8957.322128] kernel: usb 1-2.3: can't read configurations, error -32
[ 8958.003526] kernel: usb 1-2-port3: unable to enumerate USB device
```

```
[ 7280.022175] kernel: usb 1-2-port4: Cannot enable. Maybe the USB cable is bad? 
[ 7280.990299] kernel: usb 1-2-port4: Cannot enable. Maybe the USB cable is bad? 
[ 7281.960402] kernel: usb 1-2-port4: Cannot enable. Maybe the USB cable is bad?
```

**Comment by mengdonglin** [comment]
@keqiaozhang @aiChaoSONG Can you help confirm if this error related to USB sound card or USB network adaptor?

**Comment by mengdonglin** [comment]
Similar issue is observed in CI daily test ID 1828 on  2021-01-26:  on device CML_HEL_RT5682 (DUT jf-cml-hel-rt5682-02) in case check-kmod-load-unload.sh
```
kernel: [ 7293.017347] usb 1-2.4: reset full-speed USB device number 4 using xhci_hcd
kernel: [ 7294.252207] usb 1-2.4: reset full-speed USB device number 4 using xhci_hcd
kernel: [ 7294.342871] usb 1-2.4: device descriptor read/64, error 8
kernel: [ 7294.539943] usb 1-2.4: device descriptor read/64, error -32
kernel: [ 7294.734917] usb 1-2.4: reset full-speed USB device number 4 using xhci_hcd
kernel: [ 7294.824895] usb 1-2.4: device descriptor read/64, error -32
kernel: [ 7295.022892] usb 1-2.4: device descriptor read/64, error -32
kernel: [ 7295.214835] usb 1-2.4: reset full-speed USB device number 4 using xhci_hcd
kernel: [ 7295.227821] usb 1-2.4: device descriptor read/all, error -32
kernel: [ 7295.317115] usb 1-2.4: reset full-speed USB device number 4 using xhci_hcd
kernel: [ 7295.317281] usb 1-2.4: Device not responding to setup address.
kernel: [ 7295.527029] usb 1-2.4: Device not responding to setup address.
kernel: [ 7295.735821] usb 1-2.4: device not accepting address 4, error -71
kernel: [ 7295.736915] usb 1-2.4: USB disconnect, device number 4
kernel: [ 7295.861803] usb 1-2.4: new full-speed USB device number 6 using xhci_hcd
kernel: [ 7295.951888] usb 1-2.4: device descriptor read/64, error -32
kernel: [ 7296.143864] usb 1-2.4: device descriptor read/64, error -32
kernel: [ 7296.335839] usb 1-2.4: new full-speed USB device number 7 using xhci_hcd
kernel: [ 7296.423867] usb 1-2.4: device descriptor read/64, error -32
kernel: [ 7296.614059] usb 1-2.4: Device not responding to setup address.
kernel: [ 7296.822125] usb 1-2.4: Device not responding to setup address.
kernel: [ 7297.030744] usb 1-2.4: device not accepting address 7, error -71
kernel: [ 7297.031055] usb 1-2-port4: attempt power cycle
kernel: [ 7297.658831] usb 1-2.4: new full-speed USB device number 8 using xhci_hcd
kernel: [ 7297.671781] usb 1-2.4: device descriptor read/8, error -32
kernel: [ 7297.787267] usb 1-2.4: device descriptor read/all, error -32
kernel: [ 7297.877766] usb 1-2.4: new full-speed USB device number 9 using xhci_hcd
kernel: [ 7297.877975] usb 1-2.4: Device not responding to setup address.
kernel: [ 7298.101181] usb 1-2.4: device descriptor read/8, error -32
kernel: [ 7298.220439] usb 1-2.4: device descriptor read/8, error -32
kernel: [ 7298.328195] usb 1-2-port4: unable to enumerate USB device
```

**Comment by mengdonglin** [comment]
@fredoh9 Can you help check if this is related to the USB sound card or USB ethernet adaptor? Does it always happen on the same CML Helios device?

**Comment by xiulipan** [comment]
@marc-hb @aiChaoSONG It seems we missed USB-1 in https://github.com/thesofproject/sof-test/pull/568
@aiChaoSONG would  you mind to create a new PR?

**Comment by aiChaoSONG** [comment]
These usb errors are never seen when UGreen USB hub is used on helios, for this, I think we should replace usb hub with new one. I will track daily, and see if there errors are still there.

**Comment by marc-hb** [comment]
Thanks @aiChaoSONG . If some very specific USB hub or device causes error messages then we should really, really get rid of it as soon as we can replace it. We really don't want it to make the network or the audio adapter unreliable.

**Comment by fredoh9** [comment]
I replaced with known to work fine. Left is new one, right side is before. Both are from UGreen. I ran manual test, it looks working fine. I will wait and check for couple more days. If the problem is not happening again, I will close this issue. BTW, I replaced same USB hub for all Helios in JF(3 DUTs)

![20210222_142415_ugreen](https://user-images.githubusercontent.com/45636982/108784731-df444d80-7524-11eb-9d98-528bbf57e696.jpg)
